.woodpecker/docs.yml

```diff
@@ -75,7 +75,8 @@ steps:
       - git config --global user.name "woodpecker-bot"
       - git clone --depth 1 --single-branch git@github.com:woodpecker-ci/woodpecker-ci.github.io.git /repo
       # update latest and next version
-      - if [ "$CI_PIPELINE_EVENT" == "tag" ] ; then jq '.latest = ${CI_COMMIT_TAG}' /repo/version.json > /repo/version.json.tmp && mv /repo/version.json.tmp /repo/version.json ; fi
+      - if [[ "$CI_PIPELINE_EVENT" == "tag" && "${CI_COMMIT_TAG}" != *"rc"* ]] ; then jq '.latest = "${CI_COMMIT_TAG}"' /repo/version.json > /repo/version.json.tmp && mv /repo/version.json.tmp /repo/version.json ; fi
+      - if [[ "$CI_PIPELINE_EVENT" == "tag" ]] ; then jq '.rc = "${CI_COMMIT_TAG}"' /repo/version.json > /repo/version.json.tmp && mv /repo/version.json.tmp /repo/version.json ; fi
```

Comment:
So the rc channel will always be the latest release including rc and stable ones, right?

Reply:
Just checked again and the added `"` should solve the current ci issue as well:

```
+ if [ "$CI_PIPELINE_EVENT" == "push" ] ; then jq '.next = "next-0fc428aa8e"' /repo/version.json > /repo/version.json.tmp && mv /repo/version.json.tmp /repo/version.json ; fi
+ rsync -r --exclude .git --exclude CNAME --exclude index.yaml --exclude README.md --exclude version.json --delete docs/build/ /repo
jq: error: Invalid numeric literal at EOF at line 1, column 4 (while parsing '.0.0') at <top-level>, line 1:
.latest = v2.0.0            
jq: error: syntax error, unexpected LITERAL, expecting $end (Unix shell quoting issues?) at <top-level>, line 1:
.latest = v2.0.0            
jq: 2 compile errors
```